modules/ROOT/pages/design-import-files.adoc

```diff
@@ -16,8 +16,8 @@ The fragments that you import can be written in RAML 0.8 or RAML 1.0. You can im
 
 == Procedure
```

Comment:
This is quite a mixture of nested different formats. Consider reevaluating for making this easier on the audience's eyes....